test/perf/ParseBenchmarks.py

```diff
@@ -0,0 +1,264 @@
+# SPDX-License-Identifier: Apache-2.0
+
+######################################################################################################
+# There are four possible flags to add to this script:
+#
+# --run <Op>: Compute performance benchmarks for specified Op, and write to output file
+#
+# --runall: Compute performance benchmarks for all Ops in OpsWithPerformanceBenchmarks array
+#
+# --readrun <File> <Op>: Compute performance benchmarks for specified Op, and compare with benchmarks
+# already written to specified file (File must contain the same Op)
+#
+# --compare <Op1> <Op2>: Compare performance benchmarks written to file for both specified files (each
+# should contain the same Op)
+######################################################################################################
+
+
+
+import os
+import subprocess
+import sys
+import time
+
+
+if (len(sys.argv) < 2):
+    raise RuntimeError(
+        "Specify an option: --run|--runall|--readrun|--compare"
+    )
+
+
+if (not os.environ.get('ONNX_MLIR_HOME', None)):
+    raise RuntimeError(
+        "Environment variable ONNX_MLIR_HOME is not set, please set it to the path to "
+        "the HOME directory for onnx-mlir. The HOME directory for onnx-mlir refers to "
+        "the parent folder containing the bin, lib, etc sub-folders in which ONNX-MLIR "
+        "executables and libraries can be found, typically `onnx-mlir/build/Debug`"
+    )
+
+
+# Add compatible ops here; used when script is run with '--runall'
+OpsWithPerformanceBenchmarks = [
+    'Gemm',
+    'Conv'
+]
+
+
+RuntimeEpoch = int(time.time())
+
+
+

```

Comment:
Please add a short comment to detail the info held by the result. You could also define a constant dictionary that maps the name of the metrics to the number used in the Metrics list.